x-pack/plugins/security_solution/public/common/components/event_details/get_alert_summary_rows.tsx

```diff
@@ -151,50 +129,34 @@ export const getSummaryRows = ({
   const tableFields = getEventFieldsToDisplay({ eventCategory, eventCode });
 
   return data != null
-    ? tableFields.reduce<SummaryRow[]>((acc, item) => {
-        const initialDescription = {
```

Comment:
I moved all this logic to `getEnrichedFieldInfo` to make it reusable in other places.

Reply:
renaming `item` to `field` in this `reduce` is a subtle, yet appreciated improvement to the readability when it's used below 🙏